apps/src/templates/sectionProgressV2/ProgressIcon.jsx

```diff
@@ -52,8 +52,12 @@ export default function ProgressIcon({itemType}) {
         <FontAwesome
           id={'uitest-' + itemType[0]}
           icon={itemType[0]}
-          style={{color: itemType[1]}}
-          className={styles.fontAwesomeIcon}
+          style={
+            className && itemType === ITEM_TYPE.CHOICE_LEVEL
+              ? {color: itemType[2]}
+              : {color: itemType[1]}
+          }
+          className={className ? className : styles.fontAwesomeIcon}
```

Comment:
Instead of passing in a new class name for each icon to override the styles, I think it's cleaner to just assign different styles to the existing `fontAwesomeClass` depending on its particular location. See the suggestion in the CSS file for how this could fix up the legend. You'd need to do something similar for the popup window.